src/modules/colorPicker/ColorPicker/dllmain.cpp

```diff
@@ -221,7 +221,7 @@ class ColorPicker : public PowertoyModuleIface
             // Otherwise call a custom function to process the settings before saving them to disk:
             // save_settings();
         }
-        catch (std::exception ex)
+        catch (std::exception& ex)
```

Comment:
```suggestion
        catch (std::exception&)
```

Hi, thank you for the contribution! CI has failed to build  https://dev.azure.com/ms/PowerToys/_build/results?buildId=368631&view=logs&j=0f660c0a-4423-5c5d-276b-f98b39c69e13&t=0b176b82-c97a-5033-1567-981690da77e3&l=2845

Could you please remove variable name where it's not used to fix it? Thank you.

Reply:
No problems. Tried to make as little changes as possible.
How did it pass before ?:)

Reply:
Looks like passing exception by copy triggers a false negative warning bug on the MSVC side. Might report that as well 😄